src/corehost/cli/json.h

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef __JSON_H__
+#define __JSON_H__
+
+#include <memory>
+#include "pal.h"
+#include "rapidjson/document.h"
+#include "rapidjson/fwd.h"
+
+class json_parser_t {
```

Comment:
Could we name the .h/.cpp file `json_parser` instead of just `json` since this is the only thing in it?

Reply:
Sure.